core/src/main/java/hudson/Functions.java

```diff
@@ -2297,4 +2297,13 @@ public static boolean isContextMenuVisible(Action a) {
             return true;
         }
     }
+
+    /**
+     * @return Time interval in seconds between two ajax call, minimum and default is 5.
+     * Made to be set in jelly for the script hudson-behavior.js refreshPart().
+     */
+    public static int getAjaxRefreshInterval() {
```

Comment:
```suggestion
    @Restricted(NoExternalUse.class)
    public static int getAjaxRefreshInterval() {
```

Reply:
Not even sure the `static` is required here